Getting-Started.md

```diff
@@ -398,6 +482,65 @@ starting/(re)creating/building/attaching containers for a service.
 	```
     And restart the system.
 
+
+#### SLL error [&uarr;](#ssl-)
+
+![ssl_error](https://user-images.githubusercontent.com/53489999/77805355-aea32c00-70a7-11ea-9222-5073851bdc60.png)
```

Comment:
Please follow the same format of the other troubleshooting entries: "Indications, Diagnosis and Solution"